static/styles/fec.scss

```diff
@@ -0,0 +1 @@
+@import "fec-style/scss/styles.scss";
```

Comment:
Why is this being done here rather the `/static/styles/styles.scss`.

Also, I'm getting confused when things are referencing the local styles or fec-style.